Source/Core/VideoCommon/BPFunctions.cpp

```diff
@@ -265,6 +265,14 @@ void SetScissorAndViewport(FramebufferManager* frame_buffer_manager, ScissorPos
     y = static_cast<float>(g_gfx->GetCurrentFramebuffer()->GetHeight()) - y - height;
 
   g_gfx->SetViewport(x, y, width, height, near_depth, far_depth);
+
+  g_gfx->SetViewportAndScissor(AbstractGfx::ViewportAndScissors{.scissors_rect = converted_rc,
```

Comment:
This makes it look like you set the viewport twice? I think these functions do different things? One just stores some state and one actually changes the viewport and scissor rect, or something like that? I think the function naming should make that more obvious.